dist/stat_data.py

```diff
@@ -305,6 +305,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_pages_queued_urgent_hs_dirty', 'pages queued for urgent eviction from history store due to high dirty content'),
     CacheStat('cache_eviction_queue_empty', 'eviction server candidate queue empty when topping up'),
     CacheStat('cache_eviction_queue_not_empty', 'eviction server candidate queue not empty when topping up'),
+    CacheStat('cache_eviction_restored_pos_differ', 'eviction walk restored position differs from the saved one'),
```

Comment:
I am not able get the proper use of this stat, at present we already have the evict walks completed and abandoned, this stat for me is not giving any more info for diagnostics.

Reply:
Done